Unix/http/httpclientauth.c

```diff
@@ -684,11 +684,12 @@ static char *_BuildInitialGssAuthHeader(_In_ HttpClient_SR_SocketData * self, MI
     // gss_OID_set_desc mechset_krb5 = { 1, &mech_krb5 };
     // gss_OID_set_desc mechset_iakerb = { 1, &mech_iakerb };
     const gss_OID_desc mechset_avail_elems[] = {
+        { 6, "\053\006\001\005\005\002" },
         { 10, "\x2b\x06\x01\x04\x01\x82\x37\x02\x02\x0a" }, // For now we start with ntlm
         // mech_krb5,   Not yet
         // mech_iakerb,  Not yet
     };
-    const gss_OID_set_desc mechset_avail = { /*3, */ 1, (gss_OID) mechset_avail_elems };
+    const gss_OID_set_desc mechset_avail = { 2, /* 1 ,*/ (gss_OID) mechset_avail_elems };
```

Comment:
This is littered with "magic numbers", like at line 687, which you added. Comments as to what these are would be appreciated!